pkg/obfuscator/domain.go

```diff
@@ -2,102 +2,77 @@ package obfuscator
 
 import (
 	"fmt"
-	"path/filepath"
 	"regexp"
 	"strings"
 )
 
+const (
+	domainPattern      = `([a-zA-Z0-9\.]*\.)?(%s)`
+	obfuscatedTemplate = "domain%07d"
```

Comment:
aside that we now replace it with this pattern, can't we solve the whole thing with the keyword obfuscator?

Reply:
ah no, we're doing additional logic for detecting the sub domains. nvm!